tests/genomics_tests/test_genomic_pipeline.py

```diff
@@ -1107,1865 +1123,2476 @@ def test_aw2_wgs_gc_metrics_ingestion(self):
         self.assertEqual(GenomicSubProcessResult.SUCCESS, self.job_run_dao.get(2).runResult)
         self.assertEqual("SUCCESS", self.job_run_dao.get(2).runResultStr)
 
-    def test_gc_metrics_reconciliation_vs_array_data(self):
-
-        # Create the fake ingested data
-        self._create_fake_datasets_for_gc_tests(3, arr_override=True, array_participants=[1, 2, 3],
-                                                genome_center='rdr',
-                                                genomic_workflow_state=GenomicWorkflowState.AW1)
-        bucket_name = _FAKE_GENOMIC_CENTER_BUCKET_BAYLOR
-        create_ingestion_test_file('RDR_AoU_GEN_TestDataManifestWithFailure.csv',
-                                         bucket_name,
-                                         folder=config.getSetting(config.GENOMIC_AW2_SUBFOLDERS[1]))
-
-        self._update_test_sample_ids()
-
-        self._create_stored_samples([
-            (1, 1001),
-            (2, 1002),
-            (3, 1003)
-        ])
+    # DA-2934 Reconciliation process deprecated
```

Comment:
Instead of commenting on the tests, we need to delete them as well as the corresponding logic.

Reply:
The commented out tests have been deleted.